### PR TITLE
[WOR-1609] Add info button to describe Estimated Storage Cost

### PR DESCRIPTION
**Pull request by trholdridge**
### Jira Ticket: https://broadworkbench.atlassian.net/browse/WOR-1609

There is now an info button next to the estimated cost. When selected, it displays a caveat about the cost estimation.

I also edited the left margins of other info buttons in the Workspaces dashboard to be consistent with those in Workspace Notifications.

image

### CODE REVIEW

**Review by cahrens — APPROVED** [review]
The code all looks good! For some reason it failed to create a PR deployment for you (it said there was no PR number). I am re-running the deployment build to see if that fixes it (so that reviewers can just use the PR review site instead of checking out the branch and running terra-ui locally).

You will need a second reviewer-- with terra-ui, our CODEOWNERS only assigns one user because if another team is changing of our files, they should choose a second reviewer from their own team. But since this change only impacts our team, the second reviewer will need to be from our team as well. @blakery would be a good choice.

File: src/workspaces/dashboard/CloudInformation.ts
Comment (cahrens) [inline src/workspaces/dashboard/CloudInformation.ts:175]: We _could_ only show the info icon once the estimate is available, but I don't feel that strongly about it.

Comment (trholdridge) [reply src/workspaces/dashboard/CloudInformation.ts:175]: Info icon always present:
https://github.com/user-attachments/assets/83243c48-393c-48dd-9e1f-5db10bd4b061

Info icon shows up once estimate does:
https://github.com/user-attachments/assets/7cb0fbd7-716a-438c-b7fe-2e66ed64677d

Let me know if you have a preference... I kind of like it better with the icon always there, but not super strongly.

Comment (cahrens) [reply src/workspaces/dashboard/CloudInformation.ts:175]: Either way is fine… go with keeping it always there.

**Review by blakery — APPROVED** [review]
LGTM

**Pull request merged**
